packages/eslint-config-algolia/rules/base.js

```diff
@@ -327,5 +342,43 @@ module.exports = {
         quoteProps: 'as-needed',
       },
     ],
+
+    // ---- JSDoc
+    'jsdoc/check-alignment': ['error'],
+    'jsdoc/check-examples': [
+      'error',
+      { checkEslintrc: true, exampleCodeRegex: '```js ([\\s\\S]*)```' },
+    ],
+    'jsdoc/check-indentation': ['off'],
+    'jsdoc/check-param-names': [
+      'error',
+      {
+        checkDestructured: true,
+        allowExtraTrailingParamDocs: true,
+      },
+    ],
+    'jsdoc/check-syntax': ['error'],
+    'jsdoc/check-tag-names': [
+      'error',
+      {
+        definedTags: ['required', 'swagger'],
+      },
+    ],
+    'jsdoc/check-types': ['error'],
+    'jsdoc/implements-on-classes': ['error'],
+    'jsdoc/match-description': ['error'],
+    'jsdoc/newline-after-description': ['error'],
+    'jsdoc/no-types': ['error'],
```

Comment:
should this maybe only be used in typescript?

Reply:
indeed 👌🏻